pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -376,6 +378,17 @@ func (r *rbac) handleSCCInNamespace(ctx context.Context, ns *corev1.Namespace) e
 	return nil
 }
 
+func getRBACMaxCalls() int {
+	envValue := os.Getenv("OCP_RBAC_MAX_CONCURRENT_CALLS")
+	if envValue == "" {
+		return defaultRbacMaxConcurrentCalls
+	}
+	if parsedValue, err := strconv.Atoi(envValue); err == nil {
+		return parsedValue
```

Comment:
@jkhelil please do some range verification, for example: allow only `1` to `100`. 
do not allow, `0`, `-1`, `1000`, etc.,